js/compiler.ts

```diff
@@ -196,7 +196,10 @@ export class DenoCompiler
    * Get the dependencies for a given module, but don't run the module,
    * just add the module factory to the run queue.
    */
-  private _gatherDependencies(moduleMetaData: ModuleMetaData): void {
+  private _gatherDependencies(
+      moduleMetaData: ModuleMetaData,
+      forceCompilation?: boolean
```

Comment:
`s/forceCompilation/recompile/` ? Just to stick with the same nomenclature?

Reply:
I got rid of that argument entirely.